### PR TITLE
Da respondent

### PR DESCRIPTION
**Pull request by sealDebolina**
**Before creating a pull request make sure that:**
refactoring DA respondent tasklist to common tasklist

### Change description ###

https://tools.hmcts.net/jira/browse/PRL-4430

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/steps/prl-cases/dashboard/tabView.ts [hunk 1]
```diff
@@ -273,11 +266,7 @@ const getTaskListUrl = (
   let url;
 
   if (linkPartyType === PartyType.RESPONDENT) {
-    if (caseType === CaseType.C100) {
-      url = applyParms(`${FETCH_CASE_DETAILS}`, { caseId: caseNumber });
-    } else {
-      url = `${RESPONDENT_TASK_LIST_URL}/${caseNumber}`;
-    }
+    url = applyParms(`${FETCH_CASE_DETAILS}`, { caseId: caseNumber });
```
Comment (vivek-sekhar) [inline src/main/steps/prl-cases/dashboard/tabView.ts:269]: refactor this as discussed

File: src/main/routes.ts [hunk 2]
```diff
@@ -151,7 +151,8 @@ export class Routes {
     app.get(TESTING_SUPPORT_DELETE_DRAFT, errorHandler(new DeleteDraftGetController().get));
     app.get(ACCESSIBILITY_STATEMENT, errorHandler(new AccessibilityStatementGetController().get));
     app.get(CONTACT_US, errorHandler(new ContactUsGetController().get));
-    app.get(`${APPLICANT_TASK_LIST_URL}/:caseId`, errorHandler(new GetCaseController().fetchAndRedirectToTasklist));
+    //app.get(`${APPLICANT_TASK_LIST_URL}/:caseId`, errorHandler(new GetCaseController().fetchAndRedirectToTasklist));
+    app.get(`${APPLICANT_TASK_LIST_URL}/:caseId`, errorHandler(new CaseDetailsGetController().get));
```
Comment (vivek-sekhar) [inline src/main/routes.ts:155]: Revisit this.  Not required.

File: src/main/routes.ts [hunk 3]
```diff
@@ -168,7 +169,7 @@ export class Routes {
     );
     app.get(SAVE_AND_SIGN_OUT, errorHandler(new SaveSignOutGetController().get));
     app.get(TIMED_OUT_URL, errorHandler(new TimedOutGetController().get));
-    app.get(RESPONDENT_TASK_LIST_URL, errorHandler(new RespondentTaskListGetController().load));
+    app.get(RESPONDENT_TASK_LIST_URL, errorHandler(new CaseDetailsGetController().load));
```
Comment (vivek-sekhar) [inline src/main/routes.ts:172]: Revisit this.  Not required.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/routes.ts (hunk 3 above)
Comment (sealDebolina) [reply src/main/routes.ts:168]: will be taken care in different ticket

File: src/main/steps/prl-cases/dashboard/tabView.ts (hunk 1 above)
Comment (sealDebolina) [reply src/main/steps/prl-cases/dashboard/tabView.ts]: done

**Review by vivek-sekhar — APPROVED** [review]
Need to address below items in the next refactoring task,
1. Revisit APPLICANT_TASK_LIST_URL, RESPONDENT_TASK_LIST_URL & ${RESPONDENT_TASK_LIST_URL}/:caseId in routes
2. Revisit src/main/steps/respondent/task-list/index.ts.
3. Revisit src/main/steps/applicant/task-list/index.ts.

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**